src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -225,6 +237,84 @@ export default {
 			// 		},
 			// 	});
 			// }
+		},
+		fetchRemainingLoanChannels() {
+			const ssrLoanIds = [];
+			// pick loan ids from each
+			_each(this.categories, category => {
+				_each(category.loans.values, loan => {
+					ssrLoanIds.push(loan.id);
+				});
+			});
+			// Client Fetch the remaining category rows
+			return this.apollo.query({
+				query: loanChannelQuery,
+				variables: {
+					ids: _drop(this.realCategoryIds, ssrRowLimiter),
+					excludeIds: ssrLoanIds,
+					// @todo variables for fetching data for custom channels
+				},
+			}).then(({ data }) => {
+				// add our remaining loan channels
+				this.clientCategories = _get(data, 'lend.loanChannelsById') || [];
+				Promise.resolve();
```

Comment:
was this meant to be `return Promise.resolve();`?

Reply:
I just want to make sure this returns and completes the promise... guess I should just 'return someData'...